lib/client.js

```diff
@@ -15,7 +15,7 @@ var Client = function () {};
 var defaultOpt = {
   port: 30080,
   sendBounds: true,
-  verbose: false
+  verbose: 1
```

Comment:
would you like to continue keep backward compatibility with verbose like I had already implemented?

or just remove verbose all together?